run-checks

```diff
@@ -134,27 +134,46 @@ if [ "$STATIC" = 1 ]; then
     # check commit author/committer name for unicode
     ./check-commit-email.py
 
-    if [ -z "${SKIP_GOFMT:-}" ]; then
-        echo Checking formatting
-        fmt=""
-        for dir in $(go list -f '{{.Dir}}' ./... | grep -v '/\(c-\)\?vendor/' ); do
-            # skip vendor packages
-            s="$(${GOFMT:-gofmt} -s -d "$dir" || true)"
-            if [ -n "$s" ]; then
-                fmt="$s\\n$fmt"
-            fi
-        done
-        if [ -n "$fmt" ]; then
-            echo "Formatting wrong in following files:"
-            # shellcheck disable=SC2001
-            echo "$fmt" | sed -e 's/\\n/\n/g'
-            exit 1
-        fi
+    # golangci's output isn't as clear when there are compilation errors
+    go build github.com/snapcore/snapd/...
+
+    if ! command -v golangci-lint >/dev/null; then
+      # go get (with go 1.13) is killed by OOM killer
+      curl -s -XGET -L https://github.com/golangci/golangci-lint/releases/download/v1.40.1/golangci-lint-1.40.1-linux-amd64.tar.gz --output golangci.tar.gz
```

Comment:
while at it, maybe update the version to 1.44.2 which is the latest, both here and in the workflow